tests/unit_tests/db_engine_specs/test_doris.py

```diff
@@ -145,3 +151,52 @@ def test_get_schema_from_engine_params() -> None:
         )
         is None
     )
+
+
+def test_get_default_catalog() -> None:
+    """
+    Test the ``get_default_catalog`` method.
+    """
+    from superset.db_engine_specs.doris import DorisEngineSpec
+    from superset.models.core import Database
+
+    database = Mock(spec=Database)
+
+    # Test with catalog.schema format
+    database.url_object.database = "catalog1.schema1"
+    assert DorisEngineSpec.get_default_catalog(database) == "catalog1"
+
+    # Test with only catalog format
+    database.url_object.database = "catalog1"
+    assert DorisEngineSpec.get_default_catalog(database) == "catalog1"
+
+    # Test with None
+    database.url_object.database = None
+    assert DorisEngineSpec.get_default_catalog(database) is None
```

Comment:
Could we parametrize this test with `pytest.mark.parametrize`?

Reply:
Thanks, I have changed it as requested.